NYTProf.xs

```diff
@@ -1748,7 +1748,7 @@ DB_leave(pTHX_ OP *op, OP *prev_op)
 static void
 set_option(pTHX_ const char* option, const char* value)
 {
-    if (!value || !*value)
+    if (!option || !*option)
```

Comment:
This one has me a little confused. I see this is fixing a redundant check on `value` and a missed check on `option`. The change does seem to correct a required precondition. Just wanted to make sure I understood the actual reason for change. It's pretty obvious, but also makes me wonder what that actual original intent was.

Reply:
Sounds like we have to do some git archeology here.

Reply:
You might be correct -- or might not.  Here's where the code entered the file:
```
commit 60fe3f92b25896cfafa7fd1fd2b8389172bde2c9
Author: Tim Bunce <Tim.Bunce@pobox.com>
Date:   Sun Feb 24 18:22:38 2013 +0000

    Rework option handling slightly. Add option values to data file.
...

60fe3f92 (Tim Bunce            2013-02-24 18:22:38 +0000 1751)     if (!value || !*value)
60fe3f92 (Tim Bunce            2013-02-24 18:22:38 +0000 1752)         croak("%s: invalid option", "NYTProf set_option");
60fe3f92 (Tim Bunce            2013-02-24 18:22:38 +0000 1753)     if (!value || !*value)
60fe3f92 (Tim Bunce            2013-02-24 18:22:38 +0000 1754)         croak("%s: '%s' has no value", "NYTProf set_option", option);
```

Reply:
It's a correct fix. There's a `if (!value || !*value)` just below it, so it was duplicated.